tests/explainers/computer_vision/object_detection/test_object_detector.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import torch
+
+from foxai.explainer.computer_vision.object_detection.base_object_detector import (
+    BaseObjectDetector,
+)
+
+
+def test_preprocessing_should_resize_and_transpose_image():
+    img = np.zeros((1, 100, 100, 3))
+    new_shape = (300, 300, 3)
```

Comment:
in the case of testing, choosing a rectangle shape, not a square, is better. It is very easy to mistake height for width. If we use rectangle resizing, we can verify, that we did;t make a mistake.

Reply:
I've changed to a rectangular shape: https://github.com/softwaremill/FoXAI/pull/75/commits/671a6a6ed5825bf3a7cde4d8ca48baf5f592aaf1.